ctapipe/image/muon/intensity_fitter.py

```diff
@@ -194,6 +191,10 @@ def image_prediction(
         max_lambda_m=max_lambda.to_value(u.m),
     )
 
+# Define numba vectorized gaussian cdf
```

Comment:
Remove this and add a proper docstring to the function.

See: https://numpydoc.readthedocs.io/en/latest/format.html